tests/integration/test_sca/data/test_cases/cases_validate_remediation.yaml

```diff
@@ -0,0 +1,25 @@
+- name: Fix Failing SCA check - Pass
+  description: Validate Failing check changes to Pass after remediation
+  configuration_parameters:
+    ENABLED: 'yes'
+    INTERVAL: 5
+    POLICY_FILE: cis_centos8_validate_remediation.yaml
+  metadata:
+    policy_file: cis_centos8_validate_remediation.yaml
+    check_id: 1
+    perms: 0222
+    initial_result: failed
+    final_result: passed
+
+- name: Fix Failing SCA check - Pass
+  description: Validate Failing check changes to Pass after remediation
+  configuration_parameters:
+    ENABLED: 'yes'
+    INTERVAL: 5
+    POLICY_FILE: cis_centos8_validate_remediation.yaml
+  metadata:
+    policy_file: cis_centos8_validate_remediation.yaml
+    check_id: 2
+    perms: 0222
+    initial_result: passed
+    final_result: failed
```

Comment:
- Name: Changed in https://github.com/wazuh/wazuh-qa/pull/3653/commits/b0d84264f9ff5bfe5b5c7de1c2d114f22bd29f48
- Description: Changed in https://github.com/wazuh/wazuh-qa/pull/3653/commits/b0d84264f9ff5bfe5b5c7de1c2d114f22bd29f48
- Cases: The permission used is the same in both because of the SCA rules checks and the expected results:
   - The first check expects permissions `0222` but actual permissions are `0644`, so it fails. By changing to `0222`, it now passes. This is the expected result for this case.
   - The second check is in reverse, It expects `0644` and starting permissions are `0644`, so changing it to `0222` causes it now to fail, as this test case expects.